pkg/admission/quota/clusterresourcequota/admission.go

```diff
@@ -93,12 +93,12 @@ func (q *clusterQuotaAdmission) Validate(ctx context.Context, a admission.Attrib
 	// to its name. Namespaces are cluster level object that shouldn't go into this plugin or it get blocked listing
 	// the namespace that is just being created.
 	obj := a.GetObject()
-	objMeta, ok := obj.(metav1.ObjectMetaAccessor)
-	if !ok {
-		klog.Warningf("ClusterQuotaAdmission received non-meta object %T", obj)
+	objMeta, err := metav1.Accessor(obj)
+	if err != nil {
+		klog.V(5).Infof("ClusterQuotaAdmission received non object %T: %v", obj, err)
```

Comment:
could you bring back the previous log level (Warning) and try to test it?

Reply:
I have tested it with the warn level and could not reproduce it. The old version was reproducable very easily.

I suppose we could keep the warn log level since the error should be unlikely occurrence.